.github/workflows/docker-image.yml

```diff
@@ -2,48 +2,102 @@ name: Docker Image CI
 
 on:
   push:
-    branches: [ release/* ]
+    branches: ["**"]
   create:
     tags: "**"
 jobs:
   acas:
     runs-on: ubuntu-latest
-    strategy:
```

Comment:
What was the significance of removing this?

Reply:
> When jobs.<job_id>.strategy.fail-fast is set to true, GitHub cancels all in-progress jobs if any matrix job fails. Default: true

It was basically unnecessary to set it because we weren’t using matrix here.